packages/application-components/src/utils/deprecate-component.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import { getDisplayName } from 'recompose';
+import warning from 'warning';
+
+export default function({ message = '' }) {
```

Comment:
why is this here?
Is the intention then to import from app-kit when using `deprecate-component` in the future?